recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,250 @@
+            "iot_data",
+            "iot_jobs_data",
+            "iot",
+            "kafka",
+            "kinesis",
+            "kinesisanalytics",
+            "kinesisvideo",
+            "kms",
+            "lambda",
+            "lex",
+            "lightsail",
+            "logs",
+            "machinelearnings",
+            "macie",
+            "marketplace_entitlement",
+            "marketplacecommerceanalytics",
+            "mediaconvert",
+            "medialive",
+            "mediapackage",
+            "mediastore",
+            "mediatailor",
+            "meteringmarketplace",
+            "mobileanalytics",
+            "monitoring",
+            "mq",
+            "mturk_requester",
+            "neptune",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "pinpoint",
+            "polly",
+            "pricing",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "redshift",
+            "recognition",
+            "resource_groups",
+            "robomaker"
+            "route53",
+            "route53domains",
+            "s3",
+            "sagemaker",
+            "sdb",
+            "serverlessrepo"
+            "servicecatalog",
+            "servicediscovery",
+            "shield",
+            "signer",
+            "sms",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "text_to_speech",
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
+    options = {
+            **{ x: [True, False] for x in sdks },
+            **{
+                "shared": [True, False],
+                "fPIC": [True, False],
+                "min_size": [True, False]
+                }
+            }
+    default_options = {
+            **{ x: False for x in sdks },
+            **{
+                "shared": False,
+                "fPIC": True,
+                "min_size": False
+                }
+            }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("aws-c-common/0.4.25")
+        self.requires("aws-c-event-stream/0.1.5")
+        self.requires("zlib/1.2.11")
+        if self.settings.os != "Windows":
+            if self.settings.os != "Macos":
+                self.requires("openssl/1.1.1g")
+            self.requires("libcurl/7.71.1")
```

Comment:
This is very difficult to read

```suggestion
        if self.settings.os != "Windows":
            self.requires("libcurl/7.71.1")
        if not self.settings.os in ["Windows", "Macos"]:
            self.requires("openssl/1.1.1i")
```